src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -13,7 +13,33 @@
 import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
 
 /**
- * Print reads from a SAM/BAM/CRAM file.
+ * Copy reads in a SAM/BAM/CRAM file verbatim to a SAM file
+ *
+ * <p>
+ *     This tool copies reads from a SAM/BAM/CRAM file to a SAM file. If you are interested in printing reads in
```

Comment:
It might be useful to explain what this tool is useful for, I.e. selecting parts intervals from a bam or applying filters to a bam in order to produce a new bam that doesn't have certain classes of reads.